seneca/seneca.d.ts

```diff
@@ -3,229 +3,227 @@
 // Definitions by: Peter Snider <https://github.com/psnider/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/seneca
 
+declare function SNS(options?: SNS.Options): SNS.Instance;
```

Comment:
It's fine to add a merged `function`/`namespace`, but you should leave them inside `declare module "seneca"` so that you don't pollute the global namespace. (As long as seneca is a Node-only library; is that correct?)